libs/blocks/library-config/lists/blocks.js

```diff
@@ -283,7 +284,7 @@ export default async function loadBlocks(blocks, list, query, type) {
         const containerHtml = getHtml(container, block.path);
         e.target.classList.add('copied');
         setTimeout(() => { e.target.classList.remove('copied'); }, 3000);
-        const blob = new Blob([`${BLOCK_SPACING}${containerHtml}${BLOCK_SPACING}`], { type: 'text/html' });
+        const blob = new Blob([`${containerHtml}`], { type: 'text/html' });
```

Comment:
Does this still need to be stringified, or can this just be 
```suggestion
        const blob = new Blob([containerHtml], { type: 'text/html' });
```